src/main/java/com/echobox/api/linkedin/client/DefaultLinkedInClient.java

```diff
@@ -217,7 +223,9 @@ public Response makeRequest() throws IOException {
   @Override
   public <T> T publish(String connection, Class<T> objectType, Object jsonBody,
       List<BinaryAttachment> binaryAttachments, Parameter... parameters) {
-    verifyParameterPresence("connection", connection);
+    if (binaryAttachments == null || binaryAttachments.isEmpty()) {
```

Comment:
Why do we have this check?

Reply:
I've removed the check as it's no longer needed